README.md

```diff
@@ -50,6 +50,35 @@ Documentation for older versions or branches of s2n-tls can be generated locally
 
 Doxygen installation instructions are available at the [Doxygen](https://doxygen.nl/download.html) webpage.
 
+## Platform Support
+
+We’ve listed the distributions and platforms under two tiers: Tier 1 platforms are guaranteed to build, run, and pass tests in CI. Tier 2 platforms are guaranteed to build and we'll address issues opened against them, but they aren't currently running in our CI and are not actively reviewed with every commit. If you use a platform not listed below and would like to request (or help!) add it to our CI, please open an issue for discussion.
+
+### Tier 1
+
+|Distribution in CI	                                    |Platforms |
+|-------------------------------------------------------|---------|
+|Ubuntu18/24**                                          | x86_64  |
```

Comment:
I wonder if the "**" here looks like it applies to both 18 and 24, rather than just 24? It seems much clearer for AL2023 due to the comma + whitespace, so maybe you either need to use commas instead of "/", or also put 24 on its own line.